src/Components/Common/VerticalCard.jsx

```diff
@@ -0,0 +1,28 @@
+import './VerticalCard.css';
+
+function VerticalCard() {
+	return (
+		<div class="card-container">
+			<div class="img-box u-margin-bottom-small">
+				<img
+					src="https://i.ytimg.com/an_webp/7jaMJGtAV9M/mqdefault_6s.webp?du=3000&sqp=CP3WpZMG&rs=AOn4CLAgii4tSixi1pdbfHWD2_jra2WWlQ"
+					alt=""
```

Comment:
would suggest adding value for alt attribute since it provides alternative information for an image and is required for img tag.